cmd/prom-migrator/main.go

```diff
@@ -213,6 +218,7 @@ func parseFlags(conf *config, args []string) {
 	flag.StringVar(&conf.readerClientConfig.OnErrStr, "reader-on-error", "abort", "When an error occurs during read process, how should the reader behave. "+
 		"Valid options: ['retry', 'skip', 'abort']. "+
 		"See 'reader-on-timeout' for more information on the above options. ")
+	flag.StringVar(&conf.readerMetricsMatcher, "reader-metrics-matcher", `{__name__=".*"}`, "Metrics vector selector to read data for migration.")
```

Comment:
This should be a regex 😄 

```suggestion
	flag.StringVar(&conf.readerMetricsMatcher, "reader-metrics-matcher", `{__name__=~".*"}`, "Metrics vector selector to read data for migration.")
```

Reply:
Also, please add this flag in the `prom-migrator` readme, which is the reference doc for CLI. You can find that in `cmd/prom-migrator/README.md`.

Reply:
Also, the main_test.go will need to be fixed. Let me know if you want any help there.